apis/logging/v1/input_receiver_types.go

```diff
@@ -15,14 +25,18 @@ const (
 
 // HTTPReceiver receives encoded logs as a HTTP endpoint.
 type HTTPReceiver struct {
-	// Port the Service and the HTTP listener listen on.
-	// +kubebuilder:default:=8443
-	// +optional
-	Port int32 `json:"port"`
-
 	// Format is the format of incoming log data.
 	//
 	// +kubebuilder:validation:Enum:=kubeAPIAudit
 	// +required
 	Format string `json:"format"`
 }
+
+// SyslogReceiver receives logs from rsyslog
+type SyslogReceiver struct {
+	// The protocol of the connection the Receiver will listen on: tcp or upd
```

Comment:
syslog receiver needs a port - it does have this in common with HTTP, but  we can't assume a port is a  common requirement for all future receivers.